install.sh

```diff
@@ -21,12 +21,17 @@ else
 fi
 
 # Stay backwards compatible with older versions that had the version in the file name.
-if [[ "v1v2v3" == *"$VERSION"* ]]; then
+if [[ "v1v2v3v4" == *"$VERSION"* ]]; then
   TARNAME=gql-lint-$VERSION-$PLATFORM-$ARCH.tar.gz
 else
   TARNAME=gql-lint-$PLATFORM-$ARCH.tar.gz
 fi
-URL="https://github.com/sketch-hq/gql-lint/releases/download/$VERSION/$TARNAME"
+
+if [[ "$VERSION" == "latest" ]]; then
+  URL="https://github.com/sketch-hq/gql-lint/releases/latest/download/$VERSION/$TARNAME"
```

Comment:
Sure this works?  with latest and $version in the path?

Reply:
ugh, pushed a fix. This is the problem with not being able to fully test it locally 😅